README.md

```diff
@@ -268,6 +268,22 @@ if err := k.Load(s3.Provider(s3.Config{
 }
 ```
 
+### Reading from an AWS Secrets Manager
+
+```go
+// Load secrets in JSON string from AWS Secrets Manager.
+s, err := Provider(Config{
+SecretName: os.Getenv("AWS_REGION"),,
```

Comment:
Can we indent this, does not seem to be rendering correctly on https://github.com/missedone/koanf/tree/feat/awssm